### PR TITLE
feat(authoring): Implemented opening full-preview in new tab.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-210

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1: (e2e test would be nice)

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -2518,6 +2533,26 @@
...
+    function FullPreviewDirective(api) {
+        return {
+            scope: {
+                item: '=',
+                itemId: '@',
+                closeAction: '='
+            },
+            templateUrl: 'scripts/superdesk-authoring/views/full-preview.html',
+            link: function(scope, elem) {
+                if (scope.itemId) {
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/authoring.js]: is there any use for this one?

**Comment by petrjasek** [comment]
in full preview there are a few strings without translate annotation, otherwise looks ok

File: client/app/scripts/superdesk-authoring/authoring.js (hunk 1 above)
Comment (eozan) [inline client/app/scripts/superdesk-authoring/authoring.js]: It makes it possible to preview with only id, but currently not used.

**Comment by eozan** [comment]
Translations are fixed.

**Comment by vied12** [comment]
:+1:

**Pull request merged**